app/Views/GlobalGui.js

```diff
@@ -52,8 +52,133 @@ define(['backbone', 'JST', 'q', 'bootstrap', 'app'], function(Backbone, template
                 app.growl('Launching');
             });
 
+            // sketch of approach
+
+            // for right now, setting the APM severity to have a one-to-one with growl type
+            // growl types:
+            //      warning, success, danger, error
+            var SEVERITY_HIGH = 'danger';
+            var SEVERITY_MEDIUM = 'warning';
+            var SEVERITY_LOW = 'success';
+
+            // these were pulled from the APM source wherever "gcs_send_text_P" was used to send a STATUSTEXT
+            var statustextRegex = {
+                "ARMING MOTORS" : {severity: SEVERITY_HIGH},
+                "AUTO triggered off" : {severity: SEVERITY_LOW},
+                "Arm: Alt disparity" : {severity: SEVERITY_HIGH},
+                "Arm: Gyro cal failed" : {severity: SEVERITY_HIGH},
+                "Arm: Leaning" : {severity: SEVERITY_HIGH},
+                "Arm: Mode not armable" : {severity: SEVERITY_HIGH},
+                "Arm: Rotor not spinning" : {severity: SEVERITY_HIGH},
+                "Arm: Safety Switch" : {severity: SEVERITY_HIGH},
+                "Arm: Thr below FS " : {severity: SEVERITY_HIGH},
+                "AutoTune: Failed" : {severity: SEVERITY_HIGH},
+                "AutoTune: Started" : false, //{severity: SEVERITY_HIGH},
+                "AutoTune: Stopped" : false, //{severity: SEVERITY_HIGH},
+                "AutoTune: Success" : false, //{severity: SEVERITY_HIGH},
+                "Beginning INS calibration" : false, //{severity: SEVERITY_MEDIUM},
+                "Calibrating barometer" : false, //{severity: SEVERITY_LOW},
+                "Crash: Disarming" : {severity: SEVERITY_HIGH},
+                "DCM bad heading" : {severity: SEVERITY_HIGH},
+                "DISARMING MOTORS" : {severity: SEVERITY_HIGH},
+                "Demo Servos" : false, //{severity: SEVERITY_LOW},
+                "Disable fence failed" : {severity: SEVERITY_HIGH},
+                "EKF variance" : {severity: SEVERITY_HIGH},
+                "ERASING LOGS" : false, //{severity: SEVERITY_LOW},
+                "Enable fence failed" : {severity: SEVERITY_HIGH},
+                "Erasing logs" : false, //{severity: SEVERITY_LOW},
+                "Failsafe - Long event on" : {severity: SEVERITY_LOW},
+                "Failsafe - Short event off" : {severity: SEVERITY_LOW},
+                "Failsafe - Short event on" : {severity: SEVERITY_LOW},
+                "Fence disabled" : {severity: SEVERITY_HIGH},
+                "Fence enabled" : {severity: SEVERITY_HIGH},
+                "GROUND START" : {severity: SEVERITY_LOW},
```

Comment:
For this one, see issue #550.  I'd like to see a new field in the JSON blob that adds `message` so we can override the APM with something less confusing in some cases.  Use the APM default unless `message` is specified, then, use that.